FirebaseFunctions/Sources/Functions.swift

```diff
@@ -369,23 +376,23 @@ enum FunctionsConstants {
               appCheck: appCheck)
   }
 
-  func urlWithName(_ name: String) -> String {
+  func functionURL(forName name: String) -> URL? {
```

Comment:
small nit: I think it may be slightly more idiomatic to instead do

```suggestion
  func functionURL(for name: String) -> URL? {
```

Reply:
I initially used `forName` to compensate for `String`’s ambiguity, but it seems like all call sites make it clear what kind of string goes there. Renamed it.